runtime/compiler/control/JITaaSCompilationThread.hpp

```diff
@@ -135,6 +135,9 @@ class ClientSessionData
    void decNumActiveThreads() { --_numActiveThreads;  }
    void printStats();
 
+   void markedForDeletion() { _markedForDeletion = true;}
```

Comment:
Please change the name to setMarkedForDeletion() or markForDeletion(). The current name suggests a query.